markets/perps-market/contracts/modules/PerpsAccountModule.sol

```diff
@@ -100,6 +100,8 @@ contract PerpsAccountModule is IPerpsAccountModule {
         FeatureFlag.ensureAccessToFeature(Flags.PERPS_SYSTEM);
         Account.exists(accountId);
 
+        AsyncOrder.checkPendingOrder(accountId);
```

Comment:
Given this validation check is critical to the market, I recommend adding a single line comment that briefly informs the reader of its importance.